var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -91,6 +95,15 @@ def configure_args(self):
         if '+perl' in spec:
             args.append('PERL={0}'.format(spec['perl'].command.path))
 
+        if '+nls' in spec:
+            args.extend([
+                'LDFLAGS=-L{0}'.format(spec['gettext'].prefix.lib),
```

Comment:
```suggestion
                'LDFLAGS=' + spec['gettext'].libs.search_flags,
```
Libraries may be in `prefix.lib`, `prefix.lib64` or some weird RHEL path if using an external lib, this should be more reliable to determine the correct location.

Reply:
Oh, this is awesome!!! Will have to familiarize myself further with the rest of `filesystem.py`.